api/src/neo4j/queries/randomComponents.js

```diff
@@ -57,7 +57,8 @@ CALL apoc.cypher.run("
   MATCH (:Compartment${m} {id: $cid})-[${v}]-(:CompartmentalizedMetabolite)-[${v}]-(:Reaction)-[${v}]-(g:Gene)
   RETURN { geneCount: count(distinct g) } as data
   UNION
-  MATCH (:Compartment${m} {id: $cid})-[${v}]-(:CompartmentalizedMetabolite)-[${v}]-(:Reaction)-[${v}]-(:Subsystem)-[${v}]-(sss:SubsystemState)
+  MATCH (:Compartment${m} {id: $cid})-[${v}]-(:CompartmentalizedMetabolite)-[${v}]-(:Reaction)-[${v}]-(s:Subsystem)-[${v}]-(sss:SubsystemState)
+  USING JOIN on s
```

Comment:
It's a bit strange that I don't get any performance improvement for the compartment `cytosol`
Without `JOIN on`
```
time -p curl http://localhost:9080/api/v2/compartments/cytosol?model=HumanGem 
real 1.41
``` 
and with `JOIN on`
```
$ time -p curl http://localhost:9080/api/v2/compartments/cytosol?model=HumanGem > t1.txt
real 1.50
```

Reply:
This only applies for the `api/v2/random-components` endpoint.